torch_geometric/data/graph_store.py

```diff
@@ -58,30 +61,39 @@ class EdgeLayout(Enum):
 class EdgeAttr(CastMixin):
     r"""Defines the attributes of an :obj:`GraphStore` edge."""
 
-    # The type of the edge
-    edge_type: Optional[EdgeType]
+    # The type of the edge:
+    edge_type: EdgeType
```

Comment:
I assume this was originally optional to account for non hetero graphs. Do we have a new convention for standard graphs to define what EdgeType should be?

Reply:
I see. Need to think a bit more about this but I assume we can simply override `EdgeAttr` for the homogeneous case.

Reply:
What do you mean override for the homogenous case? We'll still need the layout info right?

Reply:
In the homogeneous case, we can override `EdgeAttr` to not include the `edge_type` information. Let me see if that works :)

Reply:
Oh I see what you mean. That makes sense.